pkg/portercontext/context.go

```diff
@@ -453,7 +452,10 @@ func (c *Context) CopyDirectory(srcDir, destDir string, includeBaseDir bool) err
 		}
 
 		if info.IsDir() {
-			return errors.WithStack(c.FileSystem.MkdirAll(dest, info.Mode()))
+			err := c.FileSystem.MkdirAll(dest, info.Mode())
+			if err != nil {
+				return fmt.Errorf("%w", err)
```

Comment:
👀 We can probably do:
```suggestion
				return err
```

Reply:
⚠️ **FIXME**, I think we have to return here. Note to myself: add a `return nil` after the error check, in `IsDir()`.

Reply:
e16dba14

Reply:
Agreed. Anywhere I was using errors.WithStack I believe we can just go with using the original error, and can skip the %w directive.